packages/synapse-react-client/src/components/JSONArrayEditor/useParseCsv.ts

```diff
@@ -36,7 +44,7 @@ export default function useParseCsv(
           dynamicTyping: !itemsAreString,
           complete: result => {
             if (result.errors.length > 0) {
-              reject(result.errors)
+              reject(new UseParseCsvError(result.errors))
```

Comment:
New lint rule requires that `reject` is called with an object that has `Error` as a superclass.